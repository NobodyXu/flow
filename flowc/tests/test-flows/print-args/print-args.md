print-args
==

Description
===
A flow that takes the arguments passed to the flow on the command line and prints 
the first one (after the flow's source URL) out on standard output

Features Used
===
* Context Flow
* Library Functions used (`args` from `flowruntime` and `stdout` from `flowstdlib`)
* Connections between functions