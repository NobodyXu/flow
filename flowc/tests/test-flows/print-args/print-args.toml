flow = "echo-args"

[[process]]
alias = "args"
source = "lib://flowruntime/args/get"

[[process]]
alias = "print"
source = "lib://flowruntime/stdio/stdout"

[[connection]]
from = "args/1"
to = "print"