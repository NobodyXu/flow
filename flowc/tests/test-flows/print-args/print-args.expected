["file:///Users/andrew/workspace/flow/flowc/tests/test-flows/print-args/print-args.json","arg1","arg2"]
